test/request_source/test_data/test-config.yaml

```diff
@@ -0,0 +1,13 @@
+options:
+  - request_body_size: 10
+    request_headers:
+      - { header: { key: ":path", value: "/a" } }
+      - { header: { key: "foo", value: "bar" } }
+      - { header: { key: "foo", value: "bar2" }, append: true }
```

Comment:
Ah, I see this append here, but it's ignored in the file backed implementation. That may be pre-existing, not sure if the gRPC request source variant that exists today minds this field.

Reply:
I'll remove this.